src/Spago/Config.purs

```diff
@@ -410,6 +391,23 @@ rootPackageToWorkspacePackage { rootPackage, workspaceDoc } = do
   hasTests <- liftEffect $ FS.exists "test"
   pure { path: "./", doc: workspaceDoc, package: rootPackage, hasTests }
 
+workspacePackageToLockfilePackage :: WorkspacePackage -> Tuple PackageName Lock.WorkspaceLockPackage
+workspacePackageToLockfilePackage { path, package } = Tuple package.name
+  { path
+  , dependencies: package.dependencies
+  , test_dependencies: foldMap _.dependencies package.test
+  , needed_by: mempty -- Note: this is filled in later
+  }
```

Comment:
These functions feel like they belong to the `Lock` module, but that would create a cycle, so they are stuck here. I think it's fine, not too much code